src/EditorFeatures/Core/Implementation/RenameTracking/RenameTrackingTaggerProvider.TrackingSession.cs

```diff
@@ -179,6 +179,13 @@ public void Cancel()
                         }
                         else
                         {
+                            // We do not yet support renaming (inline rename or rename tracking) on
```

Comment:
Inline rename and rename tracking don't share the location exclusion code? (Not criticizing this change, just trying to understand)